translations/pt-br/README.md

```diff
@@ -0,0 +1,58 @@
+<div align="center">
+  <br>
+  <img alt="Open Sauced" src="https://i.ibb.co/7jPXt0Z/logo1-92f1a87f.png" width="300px">
+  <h1>🍕 Introdução ao Open Source com OpenSauced 🍕</h1>
+    <strong>O site fornece materiais do curso Introdução ao Open Source com OpenSauced.</strong>
+</div>
+<br> 
+
+Bem-vindo ao curso de Introdução ao Open Source com OpenSauced! Este curso foi projetado para fornecer uma introdução ao open source e orientá-lo no processo de contribuição para projetos de open source. 
+
+## Suporte de idioma
```

Comment:
Do we really need to specify it again?
We already have this in the dropdown menu.